### ISSUE TITLE
AD: Keyword-related Ads in Naver Searches

### ISSUE THREAD

**Issue by sharbet** [List-KR/List-KR#429]
### URL

https://m.search.naver.com/search.naver?sm=mtp_hty.top&where=m&query=%EC%95%84%EC%9D%B4%EC%98%A4%EB%8B%896

### FILTERS

Adguard Base
Adguard Mobile
Adguard Annoyances

ListKR

oisd.nl
AdguardDNS

### Browser

Mozilla Firefox, Google Chrome, Samsung Internet

### Operating System

Android

### Ad-Blocker

AdGuard for Android

### Comment

https://ibb.co/fGR8fGL

**Comment by piquark6046** [comment]
@sharbet 
Did you use [AdGuard Content Blocker](https://play.google.com/store/apps/details?id=com.adguard.android.contentblocker) which can be downloaded on Google Play Store?

If not, did you enable HTTPS filtering of AdGuard for Android?

**Comment by sharbet** [comment]
@piquark6046 
I am using Adguard for Android application itself, and I have HTTPS filtering for Adguard enabled.
If it cannot be reproduced, then I guess I will have to live with it.

**Comment by piquark6046** [comment]
Then, please send me a HAR file [^1] capturing the issue.
If you want to send privately, please use my email address.
 `piquark6046@proton.me` ([OpenPGP encryption supported](https://keys.openpgp.org/vks/v1/by-fingerprint/FE57A8DEDF68D3A6400DF26E0E5C8ADF00DDAE40))

[^1]: https://kb.adguard.com/en/android/solving-problems/har#what-are-har-files

**Comment by piquark6046** [comment]
Did you check the following items?

-  Apps Management of AdGuard
- Data-saving web browsing mode
- Filtering method of AdGuard Network section
- HTTPS allowlist
- Allowlist

**Comment by piquark6046** [comment]
@sharbet 
Please try this:
```
m.search.naver.com##section[class*="ad_"][class*="sp_"]
```

**Comment by sharbet** [comment]
![8d578ccd-79ce-42f8-b5f5-fabd237bb375_1024x492](https://user-images.githubusercontent.com/73639287/176729664-696ee4d5-359f-4178-98d9-ba86d5ca710f.jpg)


![50de9e62-974f-442a-adec-59b3f87cf051_530x1024](https://user-images.githubusercontent.com/73639287/176729643-8b3febdf-9c7a-4e79-94cb-1f7a4690da5e.jpg)


Ad still showing

**Comment by piquark6046** [comment]
@sharbet Did you enable AdGuard protection and HTTPS filtering for Samsung Internet on Apps management of AdGuard?

**Comment by sharbet** [comment]
@piquark6046 
After applying the filter you've provided, `m.search.naver.com##section[class*="ad_"][class*="sp_"]`
it seems that Chromium *blocks* the ads, whereas Firefox does *not*.

Above is Firefox Nightly, and below is Bromite.
![ce1ca4e0-91f5-4a78-8bbb-1652680cc731_597x1280](https://user-images.githubusercontent.com/73639287/176825685-1d91fb69-5c4f-42a7-b908-71541fc54111.jpg)

I guess it's browser's fault. (HTTPS filtering for both browsers is *enabled*)

**Comment by piquark6046** [comment]
@sharbet 
Please check that your Firefox Nightly trusts AdGuard certificate by clicking the padlock on the address bar.
If your Firefox Nightly does not trust, please read [this document](https://kb.adguard.com/en/android/solving-problems/firefox-certificates).

**Comment by piquark6046** [comment]
`m.search.naver.com##section[class*="ad_"][class*="sp_"]` already exists in List-KR.

**Comment by sharbet** [comment]
@piquark6046 
Solved the problem. 

> @sharbet Please check that your Firefox Nightly trusts AdGuard certificate by clicking the padlock on the address bar. If your Firefox Nightly does not trust, please read [this document](https://kb.adguard.com/en/android/solving-problems/firefox-certificates).

I already had this applied.

I had to uncheck *ListKR* from *language filter section*, and manually add them with *custom filter lists* (Trusted option ON)


![9e6e92f1-d387-4d65-b3aa-6a56b5d28ca3_597x1280](https://user-images.githubusercontent.com/73639287/176903694-8c59c3a3-24e9-41aa-8ba0-71e3cf5fd32a.jpg)
![33e8b6df-d245-4b5b-96aa-3d1125ca8794_597x1280](https://user-images.githubusercontent.com/73639287/176903698-23e3df03-44a3-4518-a04c-97930656a22c.jpg)
![77739872-6f35-44c4-a9ae-b7e8e1225c73_597x1280](https://user-images.githubusercontent.com/73639287/176903700-0a8b6e52-86a5-4004-8b84-47362f71278e.jpg)


Strange thing is, Chromium (e.g. Bromite) worked fine without the filter change, but Firefox did not.
